commands/function-stats.md

```diff
@@ -1,15 +1,21 @@
-Return information about the function that's currently running.
+Return information about the function that's currently running and available execution engines.
 
-The reply consists of the following information about the running script:
+The reply is map with two keys:
 
-* **name:** the name of the function.
-* **command:** the command and arguments used for invoking the function.
-* **duration_ms:** the function's runtime duration in milliseconds.
+1. `running_script`: information about the running script.
+  If there's no in-flight function, the server replies with a _nil_.
+  Otherwise, this is a map with the following keys:
+  * **name:** the name of the function.
+  * **command:** the command and arguments used for invoking the function.
+  * **duration_ms:** the function's runtime duration in milliseconds.
+2. `engines`: this is an array of simple strings.
```

Comment:
Blanks needed for correct rendering of the list IIRC.

```suggestion
  Otherwise, this is a map with the following keys:

  * **name:** the name of the function.
  * **command:** the command and arguments used for invoking the function.
  * **duration_ms:** the function's runtime duration in milliseconds.

2. `engines`: this is an array of simple strings.
```